benchmarks/src/main/java/uk/ac/manchester/tornado/benchmarks/ComputeKernels.java

```diff
@@ -249,15 +246,12 @@ public static void mandelbrot(int size, short[] output) {
         float space = 2.0f / size;
 
         for (@Parallel int i = 0; i < size; i++) {
-            int indexIDX = i;
             for (@Parallel int j = 0; j < size; j++) {
 
-                int indexJDX = j;
-
                 float Zr = 0.0f;
                 float Zi = 0.0f;
-                float Cr = (1 * indexJDX * space - 1.5f);
-                float Ci = (1 * indexIDX * space - 1.0f);
+                float Cr = (1 * j * space - 1.5f);
```

Comment:
Could you revert to `indexJDX` and `indexIDX`? This kernel comes from AMD OpenCL examples, so the indexing is to indicate the levels of parallelism.